bot/exts/moderation/infraction/superstarify.py

```diff
@@ -128,16 +128,16 @@ async def superstarify(
 
         Alternatively, an ISO 8601 timestamp can be provided for the duration.
 
-        An optional reason can be provided. If no reason is given, the original name will be shown
-        in a generated reason.
+        An optional reason can be provided, which would be added to a message stating their old nickname
+        and linking to the nickname policy.
```

Comment:
Was there an official consensus on echoing back the user's name, or is that not a concern?

Reply:
This is referring to what will appear in the infraction history.

Reply:
Sure, but it does appear in the infraction history, DMs, and server invocation channel. I am curious if we want to be sending the old names as part of the (public-facing) embeds or not. It was a point touched upon during discussions, but not fully explored.

Reply:
infraction history and DM - yes, we want to know what the old nick was for future reference, and I don't see a reason for it to not be DMed.
For the public embed, from what I gathered people didn't seem to think it was an major issue, and most starts are applied form private channels anyway. Also in the vote it says to keep the content, and I can only assume it includes the old nick.